include/seqan3/alignment/pairwise/align_pairwise.hpp

```diff
@@ -19,8 +19,9 @@
 
 #include <meta/meta.hpp>
 
-#include <seqan3/alignment/pairwise/alignment_result.hpp>
 #include <seqan3/alignment/pairwise/alignment_configurator.hpp>
+#include <seqan3/alignment/pairwise/alignment_range.hpp>
```

Comment:
You introduce this include here but you don't change anything in the code? Is this necessary or is this part of a different commit?

Reply:
Before it was pulled in indirectly.